test/LEAF_Request_Portal_Tests/tests/api/FormStackControllerTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types = 1);
+
+use LEAFTest\LEAFClient;
+
+final class FormStackControllerTest extends DatabaseTest
+{
+    private static $client = null;
+
+    public static function setUpBeforeClass()
+    {
+        self::$client = LEAFClient::createRequestPortalClient();
+    }
+
+    protected function setUp()
+    {
+        $this->resetDatabase();
+    }
+
+    /**
+     * Tests the GET `formStack/version` endpoint.
+     */
+     public function testGetVersion() : void
+     {
+         $version = self::$client->get('?a=formStack/version');
+         $this->assertEquals(1, $version);
+     }
+
+     /**
+      * Tests the DELETE `formStack/[text]` endpoint.
+      */
+      public function testDeleteForm() : void
+      {
+        $delResponse = self::$client->Delete('?a=formStack/_form_f4688');
+        $this->assertNotNull($delResponse);
+        $this->assertEquals(true, $delResponse);
```

Comment:
There should be an additional check here to ensure it no longer exists. So try to get the form again and make sure the response is empty and/or null.